config/backend-config/types.go

```diff
@@ -88,6 +88,27 @@ type ConfigT struct {
 	Sources         []SourceT       `json:"sources"`
 	Libraries       LibrariesT      `json:"libraries"`
 	ConnectionFlags ConnectionFlags `json:"flags"`
+	Settings        SettingsT       `json:"settings"`
+}
+
+type SettingsT struct {
```

Comment:
Why do we keep creating structs with the `T` suffix? I thought we agreed that moving forward we would have avoided this.